util/project_allocation_mapper.py

```diff
@@ -67,7 +67,7 @@ def _send_allocation_decision_notification(self, charge_code, requestor_id, stat
     def _get_project_allocations(self, project, fetch_balance=True):
         balance_service = BalanceServiceClient()
         reformated_allocations = []
-        for alloc in portal_alloc.objects.filter(project_id=project.id):
+        for alloc in portal_alloc.objects.select_related('project', 'requestor', 'reviewer').filter(project_id=project.id):
```

Comment:
If you've pre-queried for allocations by this point, could you also do `project.allocations` instead of making a new query?

Reply:
yes. could do that. will fix.